graylog2-web-interface/src/components/lookup-tables/adapters/HTTPJSONPathAdapterFieldSet.jsx

```diff
@@ -17,46 +18,43 @@ class HTTPJSONPathAdapterFieldSet extends React.Component {
   state = {};
 
   onHTTPHeaderUpdate = (headers) => {
-    const config = ObjectUtils.clone(this.props.config);
-    config.headers = headers;
-    this.props.updateConfig(config);
+    const { config, updateConfig } = this.props;
+    const configChange = ObjectUtils.clone(config);
+    configChange.headers = headers;
+    updateConfig(config);
   };
 
   render() {
-    const { config } = this.props;
+    const { config, handleFormEvent, validationMessage, validationState } = this.props;
 
     return (
       <fieldset>
-        <Input type="text"
-               id="url"
-               name="url"
-               label="Lookup URL"
-               autoFocus
-               required
-               onChange={this.props.handleFormEvent}
-               help={this.props.validationMessage('url', 'The URL for the lookup. (this is a template - see documentation)')}
-               bsStyle={this.props.validationState('url')}
-               value={config.url}
-               labelClassName="col-sm-3"
-               wrapperClassName="col-sm-9" />
+        <URLWhitelistInput label="Lookup URL"
```

Comment:
When creating a new Data Adapter the whitelist check is returning errors before the field is set, but I think that should not happen.
![Kapture 2020-02-19 at 11 26 51](https://user-images.githubusercontent.com/716185/74825813-d5cd4780-530a-11ea-9e6b-b24419443969.gif)

Reply:
This is caused by the normal data adapters form validation not the whitelist check.
Data adapter validation happen on OnChange, on any of the field that need to be sent to the backend. Sending the data without any value in the other fields result in some params missing in the data that are sent causing this.

Reply:
I will create an issue for this